test/functional/ui/wildmode_spec.lua

```diff
@@ -52,6 +57,59 @@ describe("'wildmode'", function()
         :!^                       |
       ]])
     end)
+
+    it('is preserved during :terminal activity', function()
+      command('set wildmenu wildmode=full')
+      command('set scrollback=4')
+      if iswin() then
+        if helpers.pending_win32(pending) then return end
+        -- set_shell_powershell()
+        -- feed([[:terminal 1,2,3,4,5 | foreach-object -process {echo $_; sleep 0.1}]])
+      else
+        feed([[:terminal for i in $(seq 1 5000); do printf 'foo\nfoo\nfoo\n'; sleep 0.1; done<cr>]])
+      end
+      screen:sleep(50)        -- Allow some output.
+      feed([[<C-\><C-N>gg]])
+      feed([[:sign <Tab>]])   -- Invoke wildmenu.
+      screen:sleep(50)        -- Allow some output.
+      screen:expect([[
+        foo                      |
+        foo                      |
+        foo                      |
+        define  jump  list  >    |
+        :sign define^             |
+      ]])
+    end)
+
+    it('ignores :redrawstatus called from a timer #7108', function()
+      command('set wildmenu wildmode=full')
+      command([[call timer_start(10, {->execute('redrawstatus')}, {'repeat':-1})]])
+      feed([[<C-\><C-N>]])
+      feed([[:sign <Tab>]])   -- Invoke wildmenu.
+      screen:sleep(30)        -- Allow some timer activity.
+      screen:expect([[
+                                 |
+        ~                        |
+        ~                        |
+        define  jump  list  >    |
+        :sign define^             |
+      ]])
+    end)
+
+    it('with laststatus=0, :vsplit, :term #2255', function()
+      command('set laststatus=0 shell=sh')
```

Comment:
This should either be pending on Windows or have a Windows implementation, right (due to `shell=sh`)?